microcosm/tracing.py

```diff
@@ -0,0 +1,27 @@
+from jaeger_client import Config
+
+from microcosm.api import binding, defaults, typed
+
+
+@binding("tracer")
+@defaults(
+    sample_type="ratelimiting",
```

Comment:
I think this is a good default. It says, report up to 10 traces a second. We can switch between all the options here: https://www.jaegertracing.io/docs/1.12/sampling/ with the config I'm exposing.